omnigibson/scenes/traversable_scene.py

```diff
@@ -22,8 +22,6 @@ def __init__(
         num_waypoints=10,
         waypoint_resolution=0.2,
         use_floor_plane=True,
-        floor_plane_visible=True,
-        floor_plane_color=(1.0, 1.0, 1.0),
```

Comment:
@cgokmen are we still shipping our traverable scenes? would be good to actually include these in our next release again

Reply:
Agreed. @hang-yin please open a ticket for this too